### PR TITLE
Providing open in new window split buttons

### PR DESCRIPTION
**Pull request by rockstardev**
Continuation of #1265, I've done some style tweaking on multiple related pages. Here is how icon now looks in ListApps.cshtml:

image

@Kukks please review f99058a and let me know if we can simplify that method by removing `if` switches (I'm already redirecting to ViewApp the way POS was doing).

### CODE REVIEW

**Comment by rockstardev** [comment]
Man, these test are so flaky...

**Comment by NicolasDorier** [comment]
@rockstardev it is always the same that are flacky. There is CanUsePos and the invoice dropdown one. I have not figured out why.

**Comment by NicolasDorier** [comment]
Seems better, you want me to merge that first or you need to discuss with kukks before?

**Comment by rockstardev** [comment]
@NicolasDorier glad that you are signing off. 

I'll wait on @Kukks though to double check my change and remove unnecessary code.

I'll see if I can fix tests... had problem with clightning also for some reason.

**Comment by NicolasDorier** [comment]
the tests works

**Comment by NicolasDorier** [comment]
Lightning tests are only flacky in local I think this has to do with the fact I don't save their data, so when they crash, they start again with new either new key or new IP that the other node no longer recognize.

It does not happen on CI because we just created clean containers.

**Comment by rockstardev** [comment]
> Lightning tests are only flacky in local I think this has to do with the fact I don't save their data, so when they crash, they start again with new either new key or new IP that the other node no longer recognize. It does not happen on CI because we just created clean containers and volumes.

@NicolasDorier : https://circleci.com/gh/btcpayserver/btcpayserver/5737 (scroll to the bottom)

**Comment by rockstardev** [comment]
> the tests works

@NicolasDorier : https://circleci.com/gh/btcpayserver/btcpayserver/5740

Search for `BTCPayServer.Tests.ChromeTests.CanUseSSHService [FAIL]`

File: BTCPayServer/Controllers/AppsController.Crowdfund.cs [hunk 1]
```diff
@@ -170,6 +170,8 @@ public async Task<IActionResult> UpdateCrowdfund(string appId, UpdateCrowdfundVi
                 TempData[WellKnownTempData.SuccessMessage] = "App updated";
                 return RedirectToAction(nameof(UpdateCrowdfund), new { appId });
             }
+            // TODO: Check with Kukks if we can remove this tricky logic of redirecting to viewapp
```
[the remark this answers is not in the document]
Comment (rockstardev) [reply BTCPayServer/Controllers/AppsController.Crowdfund.cs]: :* Kukkie

**Pull request merged**